icpc-wf/ansible/scoreboard.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This playbook installs the static scoreboard
+
+- name: setup static scoreboard
+  hosts: scoreboard
+  vars:
+    host_type: scoreboard
+  become: yes
+  roles:
+    # - role: base_packages
```

Comment:
Can just be removed right?

Reply:
Wait no, we need these but it took a while if I kept these.